tests/integration/helpers/cluster.py

```diff
@@ -487,6 +484,19 @@ def get_client_cmd(self):
             cmd += " client"
         return cmd
 
+    # Returns the list of currently running docker containers corresponding to this ClickHouseCluster.
+    def get_running_containers(self):
+        # docker-compose names containers using the following formula:
+        # container_name = project_name + '_' + instance_name + '_1'
+        # We need to have "^/" and "$" in the "--filter name" option below to filter by exact name of the container, see
+        # https://stackoverflow.com/questions/48767760/how-to-make-docker-container-ls-f-name-filter-by-exact-name
+        filter_name = f'^/{self.project_name}_.*_1$'
+        # We want the command "docker container list" to show only containers' ID and their names, separated by colon.
+        format = '{{.ID}}:{{.Names}}'
+        containers = run_and_check(f"docker container list --all --filter name='{filter_name}' --format '{format}'", shell=True)
+        containers = dict([tuple(line.split(':', 1)) for line in containers.decode('utf-8').splitlines()])
```

Comment:
maybe just use dict comprehension?

Reply:
The simpliest way

```
{line.split(':', 1) for line in containers.decode('utf8').splitlines()}
```

doesn't work: `TypeError: unhashable type: 'list'`. It's probably because `line.split()` returns a list of two items, not `key:value`.

But

```
dict(line.split(':', 1) for line in containers.decode('utf8').splitlines())
```

works, so I've changed the code in this way.